src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,16 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int weight;
```

Comment:
Don't forget about access modifiers for your custom fields in the `Bulldozer` class. Consider using private access modifiers for fields and providing public getters and setters to access these fields.